dali/operators/erase/erase.cc

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/erase/erase.h"
+#include <memory>
+#include "dali/operators/erase/erase_utils.h"
+#include "dali/core/static_switch.h"
+#include "dali/pipeline/data/views.h"
+#include "dali/kernels/erase/erase_cpu.h"
+#include "dali/kernels/kernel_manager.h"
+
+#define ERASE_SUPPORTED_NDIMS (1, 2, 3, 4, 5)
+#define ERASE_SUPPORTED_TYPES (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, \
+                               uint64_t, int64_t, float, float16)
+
+namespace dali {
+
+DALI_SCHEMA(Erase)
+  .DocStr(R"code(Erases one or multiple regions from the image.
+
+The region is specified by an anchor (starting point) and a shape (dimensions).
+Only the relevant dimensions are specified.
+Non-specified dimensions are treated as if the whole range of the axis was provided.
+To specify multiple regions, anchor and shape represent multiple points consecutively
+(e.g. anchor=(y0, x0, y0, y1,...) and shape=(h0, w0, h1, w1,...)).
+
+Example 1:
+
+anchor=(10, 20), shape=(190, 200), axis_names="HW", input: layout="HWC", shape=(300, 300, 3), fill_value=0
```

Comment:
It's missing a dot before next sentence.

```suggestion
anchor=(10, 20), shape=(190, 200), axis_names="HW", input: layout="HWC", shape=(300, 300, 3), fill_value=0.
```
Or even:
```suggestion
anchor=`(10, 20)`, shape=`(190, 200)`, axis_names="HW", input: layout="HWC", shape=`(300, 300, 3)`, fill_value=`0`.
```
to not have the line break at the tuple.

Same below.

Reply:
When I tried using bad ticks like that the whole documentation was misformatted (don't really know why though)